conf/modules/generate_cytosure_files.config

```diff
@@ -0,0 +1,50 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Available keys to override module options:
+        ext.args            = Additional arguments appended to command in module.
+        ext.args2           = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3           = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix          = File name prefix for output files.
+        ext.when            = Conditional clause
+----------------------------------------------------------------------------------------
+*/
+
+//
+// VCF2Cytosure workflow options
+//
+
+process {
+
+    if ( params.analysis_type != "wes" && !params.skip_vcf2cytosure ) {
+
+        withName: '.*GENERATE_CYTOSURE_FILES:TIDDIT_COV_VCF2CYTOSURE' {
+            ext.args = { '-z 500' }
+            ext.prefix = { "${meta.id}_cov" }
+        }
+
+        withName: '.*GENERATE_CYTOSURE_FILES:SPLIT_AND_FILTER_SV_VCF' {
+            ext.args = { [
+                "--samples ${meta.id}",
+                '--output-type z',
+                "--exclude 'gnomad_svAF > 0.05'"
+                ].join(' ') }
+            ext.prefix = { ${meta.id} }
+        }
+
+        withName: '.*GENERATE_CYTOSURE_FILES:VCF2CYTOSURE' {
+            ext.args = { [
+                meta.sex.equals('1') ? '--sex male' : '--sex female',
```

Comment:
What would happen if we had unknown?

Reply:
If `meta.sex` is set to anything else than 1 it will default to female. I believe that is the general default of the program.